src/panels/config/integrations/ha-config-integration-page.ts

```diff
@@ -1538,7 +1543,7 @@ class HaConfigIntegrationPage extends SubscribeMixin(LitElement) {
         .integration-info.info ha-svg-icon {
```

Comment:
All `.integration-info` class styling can be removed now I think?

Reply:
It is still used for custom_integration, custom_overwrites_core, depends_on_cloud etc...